jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,545 @@
+
+    ```yaml
+    version: 2.1
+    jobs: 
+      build: 
+        docker: 
+          - image: "circleci/node:<< pipeline.parameters.image-tag >>"
+        environment: 
+          IMAGETAG: "<< pipeline.parameters.image-tag >>"
+        steps: 
+          - run: echo "Image tag used was ${IMAGETAG}"
+    parameters: 
+      image-tag: 
+        default: latest
+        type: string
+    ```
+
+You will need to declare the parameters you expect to receive from the API. In this case, under the `parameters` key, we definte an "image-tag" to be expected in the JSON payload of a POST request to the _Trigger New Pipeline_ endpoint.
+
+7. Now we can run a `curl` request that passes variables in a POST request, similar to the following:
+
+    ```sh
+    curl -u ${CIRCLECI_TOKEN}: -X POST --header "Content-Type: application/json" -d '{
+      "parameters": {
+        "image-tag": "4.8.2"
+      }
+    }' https://circleci.com/api/v2/project/gh/{YOUR_USER_NAME}/hello-world/pipeline
+    ```
+
+This concludes the end-to-end example of using the V2 API. For more detailed information about other endpoints you may wish to call, please refer to the [CircleCI API v2 Documentation]({{site.baseurl}}/api/v2/#circleci-api) for an overview of all endpoints currently available.
+
+# Additional API Use Cases
+
+Now that you have a general understanding of how the CircleCI API v2 service works through an end-to-end API example request and walkthrough, let's look at a few common tasks and operations you may perform on a regular basis when using the API. Whether you wish to return information about a job or project, or retrieve more detailed information about a project by reviewing its artifacts, the examples shown below should assist you in gaining a better understanding of how to make some API requests to the server so you can perform a deep dive into the specifics of your work.
+
+This section provides detailed information on how you can perform the following tasks and operations:
+
+* [Get project details](#get-project-details)
+* [Get job details](#get-job-details)
+* [Download artifacts](#download-artifacts)
+
+## Get Project Details
+
+You may often find that it would be helpful to retrieve information about a specific project, including the name of the organization the project belongs to, the version control system (vcs) that hosts the project, and other details. The CircleCI API enables you to return this and other information by making a single GET request to the `project/{project-slug}` endpoint by passing the `project-slug` parameter.
+
+**Note** If you would like more detailed information about a project, or simply need a refresher on the specifics of a project, please refer to the CircleCI [Projects](https://circleci.com/docs/2.0/projects/) page.
+
+### Prerequisites
```

Comment:
This section is repeated several times per use-case. I think we can move it up into a top level section, or just mention it once and say it applied to all "Additional API Use Cases"

Reply:
fixed in document